qiskit_nature/second_q/operators/bosonic_op.py

```diff
@@ -1,6 +1,6 @@
 # This code is part of a Qiskit project.
 #
-# (C) Copyright IBM 2023.
+# (C) Copyright IBM 2024.
```

Comment:
```suggestion
# (C) Copyright IBM 2023, 2024.
```

Reply:
This just one date when its just the one year, if it spans years then its the first year and last year.